server/src/utils/testUtils.js

```diff
@@ -25,8 +32,42 @@ export const createTestServer = async () => {
 export const createMockgoose = async () => {
   const mockgoose = new Mockgoose(mongoose);
   await mockgoose.prepareStorage().then(async () => {
-    await mongoose.connect(DB_URL, { useNewUrlParser: true, useCreateIndex: true });
+    await mongoose.connect(DB_URL, mongoConfig);
   });
 
   return mockgoose;
 };
+
+export const getMockPlayers = () => [1, 2, 3, 4, 5, 6].map(getMockUser);
+
+export const sortPlayersAndGetTopFive = mockPlayers => mockPlayers
+  .sort((player1, player2) => {
+    const { score: player1Score } = player1;
+    const { score: player2Score } = player2;
+    if (player1Score === player2Score) return 0;
+    if (player1Score < player2Score) return 1;
+    return -1;
+  })
+  .slice(0, 5)
+  .map((player) => {
+    const { username, score } = player;
+    return {
+      username,
+      score,
+    };
+  });
+
+export const seedDatabase = async (mockPlayers, server) => {
+  await mockPlayers.forEach(async (player) => {
```

Comment:
I am pretty sure that here await is not waiting for completion of forEach promises, as forEach doesn't return any thing. 
```suggestion
  await Promise.all(mockPlayers.map(async (player) => {await something()}))
```

Reply:
Okay, I will make this change.